spring-boot-tools/spring-boot-gradle-plugin/src/test/java/org/springframework/boot/gradle/junit/GradleCompatibilitySuite.java

```diff
@@ -39,7 +39,7 @@
 public final class GradleCompatibilitySuite extends Suite {
 
 	private static final List<String> GRADLE_VERSIONS = Arrays.asList("default", "3.4.1",
-			"3.5", "4.0-rc-1");
+			"3.5", "3.5.1", "4.0", "4.1-milestone-1");
 
```

Comment:
4.0.1 is also release so if this code is merged then that version should also be taken care